### PR TITLE
chore: update github templates

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Ticket Number
n/a

## Description of change
Update the PR template

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] I have merged commits to avoid fixing things in this PR
- [ ] New features have tests
- [x] All new and existing tests pass
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Description of change
+<!-- Please describe the change -->
+
+## Checklist
+<!-- Put an `x` in all the boxes that apply: -->
+- [ ] Requires infrastructure changes
+- [ ] I have updated the documentation accordingly
+- [ ] I have merged commits to avoid fixing things in this PR
+- [ ] New features have tests
+- [ ] All new and existing tests pass
```
Comment (a6software) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: For L13-14 we could negate the need for these items on the checklist by:

1. Ensuring the build fails if cypress or jest test fail (I think this is as-is anyway)
2. We could set a coverage threshold to catch new features needing tests 

Open to discussion on either point

Comment (MrSimonEmms) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: I like the idea of adding coverage thresholds - currently, it's all to 100% in the appeals-api, but doesn't have "all files set" so it's only 100% of all files that have 1 or more tests

I think I'll get rid of line 14. We can get rid of line 13 in future (do you want to lead that process?)

Comment (a6software) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes, I will look into it. Drives me nuts when stuff doesn't have tests tbh

Comment (MrSimonEmms) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think I :heart: you

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Pull request merged**